### PR TITLE
Add aiohttp-graphql

### PR DESCRIPTION
**Pull request by KingDarBoja**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml" 
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [ ] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

File: recipes/aiohttp-graphql/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - pip
+    - python
+    - aiohttp >=3,<4
+    - graphql-core >=2.0,<3
+    - graphql-server-core >=1.2,<2
```
Comment (beckermr) [inline recipes/aiohttp-graphql/meta.yaml]: Only things imported in the setup.py are needed in host.

**Comment by KingDarBoja** [comment]
We have to wait for graphql-server-core 1.2.0 to be available at conda, but everything else is fine 🚀

**Comment by KingDarBoja** [comment]
@conda-forge-admin, please rerender

**Comment by beckermr** [comment]
@KingDarBoja you need to add the license file. This is why the build is failing.

**Comment by KingDarBoja** [comment]
> @KingDarBoja you need to add the license file. This is why the build is failing.

The source repo does have a [LICENSE](https://github.com/graphql-python/aiohttp-graphql/blob/master/LICENSE) file and the `meta.yaml` file has it too 🤔

**Comment by beckermr** [comment]
It is the pypi package that does not. Go ahead and copy the license file into the recipe. Also make an issue on the upstream to include the license in the sdist.

**Comment by KingDarBoja** [comment]
@beckermr Thank you mate, now the CI tests were successful

**Pull request merged**